Shopify/hydrogen-v1
# Support sub request cache control header `stale-while-revalidate` everywhere

Fix: https://github.com/Shopify/custom-storefronts/issues/175

### Description

This is a workaround for supporting `stale-while-revalidate` cache control ~~on Cloudflare~~ everywhere

### How to test _on Cloudflare_

You must have:

* A Cloudflare domain (cannot be the `worker.dev` that Cloudflare gives you)
* A worker route that point to the Cloudflare domain

Once you have satisfy the above requirements, you can deploy your app with `setLoggerOptions({showCacheApiStatus: true})` and `wrangler tail -f pretty` to see the cache api logs.

### Before submitting the PR, please make sure you do the following:

- [ ] Read the [Contributing Guidelines](https://github.com/shopify/hydrogen/blob/main/docs/contributing.md)
- [ ] Update docs in this repository according to your change
- [ ] Run `yarn changeset add` if this PR cause a version bump based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
